xilem/src/view/portal.rs

```diff
@@ -8,7 +8,7 @@ use masonry::widgets;
 use crate::core::{DynMessage, Mut, ViewMarker};
 use crate::{MessageResult, Pod, View, ViewCtx, ViewId, WidgetView};
 
-/// A view which puts `child` into a scrollable region.
+/// Returns a view which puts `child` into a scrollable region.
```

Comment:
I don't think saying "returns a" is meaningfully better than "a" here. Conceptually, I think of this as having an elided `[The component for ]a view which ...`

That is, component functions never do anything other than returning a view, it's only noise to specify that the thing it does with the view is return it.